AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -4437,3 +4437,16 @@ post-gazette.com#$##toolbar > #share-bar { display: none!important; }
 ||kfund-media.com/wp-content/uploads/2018/01/580h400_banner_telegram-min.png
 ||mayu-pravo.com/share/slider/
 ||u.today/img/satoshi-twitter.png
+blog.csdn.net#$#.btn_open_app_prompt_div { display: none!important; }
+blog.csdn.net#$#.article_content { overflow: auto !important; height: auto !important; }
+blog.csdn.net#$#.btn_mod { display: none!important; }
+blog.csdn.net#$#.readall_box { display: none!important; }
+blog.csdn.net#$#.feed-Sign-span { display: none!important; }
+blog.csdn.net#$#.weixin-shadowbox { display: none!important; }
+jianshu.com#$#.call-app-btn { display: none!important; }
+jianshu.com#$#.collapse-tips { display: none!important; }
```

Comment:
Hi. All these elements used in popups, which block scrolling?

Reply:
maybe I misunderstanded, `blog.csdn.net#$#.article_content { overflow: auto !important; height: auto !important; }` and `jianshu.com#$#.collapse-free-content { overflow: auto !important; height: auto !important; }` is for scrolling and the others are normal HTML elements, just need to hide them or remove.

Reply:
Ok. Will try to explain)
For examle popup has three elements:
- popup content;
- overlay/bacground;
- style for scrolling blocking.
In this case you need thee rules:
- 2x `{ display: none!important; }`;
- 1x `{ overflow: auto !important; }`(or changed style for `position`, which used on some sites to block scrolling).
All other elements can be hidden by `##`.

Reply:
I think this time the rules are right.